config/schedule.rb

```diff
@@ -7,3 +7,8 @@
 every :day, at: '3:00 am', roles: [:job_creator] do
   rake 'clear_expired_sessions'
 end
+
+every Figaro.env.queue_health_check_frequency_seconds.to_i, roles: [:job_creator] do
+  runner 'WorkerHealthChecker.check'
+  runner 'WorkerHealthChecker.enqueue_dummy_jobs'
+end
```

Comment:
Does it make sense to load the whole Rails environment to write a crontab? Wouldn't it just be easier to grab the values from `ENV` directly?